script.cinder/addon.xml

```diff
@@ -13,4 +13,12 @@
 		<email>cinder.kodi@gmail.com</email>
                 <source>https://github.com/jpr679/stript.cinder</source>
 	</extension>
+	<extension point="kodi.context.item">
+		<menu id="kodi.core.main">
+			<item library="scriptcontextmenuoption.py">
+				<label>32004</label>
+				<visible>true</visible>
```

Comment:
This is not an acceptable condition. You need to narrow it down to something that makes sense for your addon, otherwise it's going to be shown for *every* single list item in kodi.